presto-docker-image/build-local.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+# Retrieve the script directory.
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

Comment:
In product tests we use this:
```
SCRIPT_DIR="${BASH_SOURCE%/*}"
```

(same in `presto-docker-image/build-remote.sh`)

Reply:
👍 Done.